webapp/src/main/webapp/package.json

```diff
@@ -12,47 +12,47 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.2.0",
-    "@angular/common": "~8.2.0",
-    "@angular/compiler": "~8.2.0",
-    "@angular/core": "~8.2.0",
-    "@angular/forms": "~8.2.0",
-    "@angular/platform-browser": "~8.2.0",
-    "@angular/platform-browser-dynamic": "~8.2.0",
-    "@angular/router": "~8.2.0",
-    "jquery": "^3.5.0",
-    "keycloak-js": "^20.0.2",
+    "@angular/animations": "~8.2.14",
```

Comment:
These ones seem reasonable updates. Did you get the opportunity to test them out?

Reply:
After update dependencies, I did a minimal manual check